src/XPurchase/VirtoCommerce.ExperienceApiModule.XOrder/Schemas/OrderSchema.cs

```diff
@@ -78,25 +70,13 @@ public void Build(ISchema schema)
                 })
             });
 
-            var orderConnectionBuilder = GraphTypeExtenstionHelper.CreateConnection<CustomerOrderType, object>()
-                .Name("orders")
-                .Argument<StringGraphType>("filter", "This parameter applies a filter to the query results")
-                .Argument<StringGraphType>("sort", "The sort expression")
-                .Argument<StringGraphType>("cultureName", "Culture name (\"en-US\")")
-                .Argument<StringGraphType>("userId", "")
-                .PageSize(20);
-
+            var orderConnectionBuilder = GraphTypeExtenstionHelper.CreateConnection<CustomerOrderType, object>().Name("orders").PageSize(20);
+            orderConnectionBuilder.FieldType.Arguments = AbstractTypeFactory<QueryConnectionArguments>.TryCreateInstance().AddArguments(orderConnectionBuilder.FieldType.Arguments);
```

Comment:
Hide second line with an extension. You have all data to make it.
```suggestion
            var orderConnectionBuilder = GraphTypeExtenstionHelper
                .CreateConnection<CustomerOrderType, object>()
                .Name("orders")
                .PageSize(20)
                .Arguments<QueryConnectionArguments>();
```